python/py-pip/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 PortGroup           select 1.0
 
 name                py-pip
-version             10.0.1
+version             18.0
 revision            0
```

Comment:
when updating to a new version the revision should be set to 0, which is the default. So this line can/should be removed (also below).